src/clj/qbits/spandex.clj

```diff
@@ -192,11 +192,11 @@
           (str/index-of "application/json")))
 
 (defn ^:no-doc gzipped-entity?
-   [^HttpEntity entity]
-   (some-> entity
-           .getContentEncoding
-           .getValue
-           (str/index-of "gzip")))
+  [^HttpEntity entity]
```

Comment:
These lines were updated by `lein cljfmt fix`, but are otherwise unrelated to this PR.